src/components/notification-list-item.component.js

```diff
@@ -76,6 +77,8 @@ export class NotificationListItem extends Component {
         return 'git-commit';
       case SubjectType.pull:
         return 'git-pull-request';
+      case SubjectType.release:
+        return 'tag';
       default:
         return 'issue-opened';
     }
```

Comment:
We'd better filter Release events in function `getTitleComponentProps` (just after here, but Github doesn't support to add review comments at unmodified codes). Then it can avoid exporting SubjectType.

I have to admit that this component is not implemented very straightforward. Because the navigationAction 
 prop is not used as press handler directly.

Reply:
@machour Can you have a rest during your huge refactoring? I bet you can fix this one in 5min  😏

Reply:
There you go! :D